### DISCUSSION TITLE
Multiplex Session Not Working

### DISCUSSION THREAD

**Discussion by Nzii3** [YimMenu/YimMenu#3567] [General]
The multiplex session option isnt working when enabled as host, and set to 5. Any tips to get it to work better as it says?
![image](https://github.com/user-attachments/assets/c6f37915-e99b-4f1f-8a5b-7bfa90875cdf)

**Comment by ghost** [comment]
Another stupid question. If you find it so hard to use the search function in Issues, then you don't know how to use GitHub. #3310 #3229

**Reply by Nzii3** [reply]
> Another stupid question. If you find it so hard to use the search function in Issues, then you don't know how to use GitHub. #3310 #3229

Damn you are so obsessed with me its hilarious. Go cry a river.

**Reply by ghost** [reply]
> Damn you are so obsessed with me its hilarious. Go cry a river.

I’m just irritated by people, especially frontend developers and discord managers, who can’t be bothered to search for previously mentioned issues

**Reply by Nzii3** [reply]
> > Damn you are so obsessed with me its hilarious. Go cry a river.
> 
> I’m just irritated by people, especially frontend developers and discord managers, who can’t be bothered to search for previously mentioned issues

Ok? You think I give a fuck? It’s not hard to copy and paste an answer. You’re so obsessed with me you went to my profile and start stalking my socials lmfao. Get a job.

**Reply by ghost** [reply]
> Ok? You think I give a fuck? It’s not hard to copy and paste an answer. You’re so obsessed with me you went to my profile and start stalking my socials lmfao. Get a job.

please stop flooding this repository and instead go moderate the discord server or other cancer

**Reply by Yimura** [reply]
No @Nzii3, @boogabooga841 kind of has a point. Please use the search functionality before creating a new post, creating new posts all the time for things that already exist will just pollute the search results for people that DO make use of the search functionality correctly.

**Reply by Nzii3** [reply]
> No @Nzii3, @boogabooga841 kind of has a point. Please use the search functionality before creating a new post, creating new posts all the time for things that already exist will just pollute the search results for people that DO make use of the search functionality correctly.

See I’ll listen to you, being polite and not being a dick.
